app/serializers/forecast_serializer.rb

```diff
@@ -6,30 +6,20 @@ def initialize(forecast_json)
   end
 
   def currently
-    c = forecast_json['currently']
-
-    return {
-      'time' =>              c['time'],
-      'summary' =>           c['summary'],
-      'icon' =>              c['icon'].gsub('-', '_'),
-      'temperature' =>       c['temperature'],
-      'precipProbability' => c['precipProbability'],
-      'precipIntensity' =>   c['precipIntensity'],
-      'windSpeed' =>         c['windSpeed'],
-      'windGust' =>          c['windGust'],
-      'windBearing' =>       c['windBearing']
-    }
+    hourly_in(0)
```

Comment:
I thought about doing it this way but currently and hourly[0] were sometimes a little different data from DarkSky.  Believe hourly[0] is the previous hour snapshot while currently is the exact minute of the request.  For example, my request right now, currently is 10:10 am, hourly[0] is 10:00 am.

But for our purposes, I think this is fine.

Reply:
I suppose I don't foresee needing to access that hourly[0] information